code/modules/customitems/item_defines.dm

```diff
@@ -246,7 +246,7 @@
 	new /obj/item/weapon/reagent_containers/food/drinks/cans/cola(src)
 
 
-/obj/item/device/guitar/jello_guitar //Antcolon3: Dan Jello
+/obj/item/device/instrument/guitar/jello_guitar //Antcolon3: Dan Jello
```

Comment:
@tigercat2000 @Necaladun @TullyBurnalot 

This is a pathing change to a custom item; when this is merged the database will also have to be changed.